### ISSUE TITLE
Results in Table 1

### ISSUE THREAD

**Issue by solucky-95** [yulu0724/SDC-IL#10]
Hi,

Thank you for sharing your awesome work.
I have a question about results in Table 1:
'E-Pre' is a ResNet-18 model pre-trained on ImageNet without fine-tuning, and you get 78.5% R1 on the first 100 classes (suppose you are using random seed 1).
If my understanding is right, then the results are a little confusing. I directly used the official ImageNet pre-trained ResNet-18 model of pytorch (set pretrained=True) and got 69.3% only for T1. Since I didn't train this model at all, it should be the same result as reported in Table 1.
Should I train this model myself on ImageNet? And will it get approximately 9% improvement than the official pre-trained model? It would be wonderful if you can provide this ImageNet pre-trained model.

Best regards.

**Comment by yulu0724** [comment]
Hi,

Thanks for the interest. Table 1 shows results on fine-grained datasets. On fine-grained datasets, the classes are divided into 6 tasks equally. For example R1 includes 33 classes for CUB dataset rather than 100 classes. That's probably why you got worse results than mine. Hope my answer helps.

Best,

**Comment by solucky-95** [comment]
Thanks.
